### ISSUE TITLE
NPE at org.udger.parser.UdgerParser.patchVersions(UdgerParser.java:667)

### ISSUE THREAD

**Issue by quonsto** [udger/udger-java#12]
The `org.udger.parser.UdgerParser.parseUa()`, being given an input of `Mozilla/5.0 (X11; U; Linux x86_64; cs-CZ) AppleWebKit/533.3 (KHTML, like Gecko) rekonq Safari/533.3` fails inside `org.udger.parser.UdgerParser.patchVersions()` with an NPE.

Relevant stack trace follows: 

```
java.lang.NullPointerException
	at org.udger.parser.UdgerParser.patchVersions(UdgerParser.java:667)
	at org.udger.parser.UdgerParser.clientDetector(UdgerParser.java:464)
	at org.udger.parser.UdgerParser.parseUa(UdgerParser.java:173)
```

**Comment by quonsto** [comment]
The useragent string `Mozilla/5.0 (X11; U; Linux x86_64; cs-CZ) AppleWebKit/533.3 (KHTML, like Gecko) rekonq Safari/533.3` is matched by regexp `mozilla.*applewebkit.*rekonq[\/]{0,1}([0-9a-z\.]+){0,1}.*` in current db, which I downloaded from udger.com. 
This regexp fails to match the version in the user agent string above.

Thus, the following code fails in `org.udger.parser.UdgerParser.patchVersions`:

```
            if (lastPatternMatcher.groupCount()>=1) {
               version = lastPatternMatcher.group(1);
            }
            ret.setUaVersion(version);
            ret.setUaVersionMajor(version.split("\\.")[0]);
```

**Comment by skybber** [comment]
Fixed in 1.0.6
